doc/models_library/iaf_psc_exp.rst

```diff
@@ -8,7 +8,7 @@ Description
 +++++++++++
 
 iaf_psc_exp is an implementation of a leaky integrate-and-fire model
-with exponential shaped postsynaptic currents (PSCs) according to [1]_.
+with exponential kernel postsynaptic currents (PSCs) according to [1]_.
```

Comment:
Maybe we can remove the word "kernel" altogether here (and in similar situations)?